### ISSUE TITLE
Login button is black on dark blue

### ISSUE THREAD

**Issue by awead** [psu-libraries/cho#947]
When not logged in, the login button cannot be seen very well due to the color contrast:
![Screen Shot 2019-04-10 at 10 02 14 AM](https://user-images.githubusercontent.com/312085/55885389-db4cf380-5b77-11e9-9346-1ae19695fc3c.png)

**Comment by ntallman** [comment]
Fixed with #993
